src/redisearch_rs/rqe_iterators_bencher/benches/iterators.rs

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+//! Benchmark iterators
+
+use criterion::{Criterion, criterion_group, criterion_main};
+use rqe_iterators_bencher::benchers;
+
+fn benchmark_empty(c: &mut Criterion) {
```

Comment:
Is this file the long-term benchmark suite, after we remove the C implementation?